File: frontend/bravito_app/lib/features/usuarios/domain/entities/usuario.dart
class Usuario {
  final String id;
  final String nome;
  final String username;
  final String email;
  final bool ativo;
  final List<String> perfis;

  Usuario({
    required this.id,
    required this.nome,
    required this.username,
    required this.email,
    required this.ativo,
    this.perfis = const [],
  });
}

File: backend/src/Bravito.Domain/Acesso/Usuario.cs
using System;
using System.Collections.Generic;

namespace Bravito.Domain.Acesso
{
    public class Usuario
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string KeycloakId { get; set; } = string.Empty;
        public string Nome { get; set; } = string.Empty;
        public string Username { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public bool Ativo { get; set; } = true;
        public DateTime DataCriacao { get; set; } = DateTime.UtcNow;
        public DateTime? DataAlteracao { get; set; }

        public ICollection<UsuarioPerfilAcesso> PerfisAcesso { get; set; } = new List<UsuarioPerfilAcesso>();
    }
}
